src/App.tsx

```diff
@@ -1,60 +1,126 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [userId, setUserId] = useState(0);
+  const [todoTitle, setTitle] = useState('');
+  const [currentTodos, setTodo] = useState(todos);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
It's better to name your state variables consistently. Change 'setTodo' to 'setCurrentTodos' to match the naming pattern of other state variables.
also follow convention for the boolean variable names